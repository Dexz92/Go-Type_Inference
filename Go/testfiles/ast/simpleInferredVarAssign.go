func main() {
  var i = 1;
  var b = true;
  var f = 1.0;
}
