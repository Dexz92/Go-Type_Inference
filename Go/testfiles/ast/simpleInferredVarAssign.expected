Program
  List
    FunctionDeclaration
      IdentifierDeclaration
        UnknownType
        Constant
      List
      VoidType
      Block
        List
          IdentifierDeclarationStatement
            IdentifierDeclaration
              InferredType
              Variable
            Opt
              Int
                IntType
          IdentifierDeclarationStatement
            IdentifierDeclaration
              InferredType
              Variable
            Opt
              Bool
                BoolType
          IdentifierDeclarationStatement
            IdentifierDeclaration
              InferredType
              Variable
            Opt
              Float64
                FloatType
